src/wallet.rs

```diff
@@ -118,6 +119,41 @@ where
 		res
 	}
 
+	pub(crate) fn build_payjoin_transaction(
+		&self, output_script: ScriptBuf, value_sats: u64,
+	) -> Result<Psbt, Error> {
+		let locked_wallet = self.inner.lock().unwrap();
+		let network = locked_wallet.network();
+		let fee_rate = match network {
```

Comment:
Added this because the test `cfg` flag does not work in integration tests, and the estimation is not accurate in `regtest`..

Reply:
I think its because the receiver also adds their inputs/outputs and they dont add any fees because we are using https://github.com/payjoin/rust-payjoin/blob/c036ec13ee343857651963a43bbdda3222626f89/payjoin/src/send/mod.rs#L190